devices/Si7021/Si7021.cs

```diff
@@ -118,21 +161,19 @@ private RelativeHumidity GetHumidity()
         }
 
         /// <summary>
-        /// Get Si7021 Firmware Revision
+        /// Get Si7021 firmware revision.
         /// </summary>
-        /// <returns>Firmware Revision</returns>
-        private byte GetRevision()
+        /// <returns>The FirmwareRevision.</returns>
+        private FirmwareRevision GetRevision()
```

Comment:
The pattern is to use Version rather than enum or equivalent.
This will simplify as well the implementation.
`private Version GetRevision()` and previously on the property `public Version FrimwareVersion => GetVersion()`*